SU2_CFD/src/integration/CMultiGridIntegration.cpp

```diff
@@ -598,8 +598,8 @@ void CMultiGridIntegration::SetRestricted_Solution(unsigned short RunTime_EqSyst
           }
           else {
             /*--- For stationary no-slip walls, set the velocity to zero. ---*/
-
-            sol_coarse->GetNodes()->SetVelSolutionZero(Point_Coarse);
+            su2double zero[3] = {0.0};
+            sol_coarse->GetNodes()->SetVelSolutionVector(Point_Coarse, zero);
```

Comment:
Thanks for this commit @pcarruscag . I forgot that that this had to be overloaded as well :( ... but just reusing the `SetVelSolutionVector` func is nice as well 👍